packages/client/src/orders/types/getGuestOrder.types.ts

```diff
@@ -0,0 +1,7 @@
+import type { Order } from './order.types';
+
+export type GetGuestOrder = (
+  id: string,
+  guestUserEmail: string,
+  config?: Record<string, unknown>,
```

Comment:
Use `Config` type here instead of `Record<string, unknown>`

Reply:
fixed